LICENSE

```diff
@@ -1,6 +1,6 @@
 Software License Agreement (BSD License)
 
- Copyright (c) 2017-2018, Epistímio.
+ Copyright (c) 2017-2020, Epistímio.
```

Comment:
I think this would be too much overheard for what it's worth. The coverage configuration was worth it because it affects the coverage measure so factoring out to another PR makes it possible to distinguish coverage of the PR properly.

Reply:
Fair enough